python_modules/libraries/dagster-databricks/dagster_databricks/databricks.py

```diff
@@ -173,14 +173,18 @@ def submit_run(self, run_config, task):
         # since they're imported by our scripts.
         # Add them if they're not already added by users in config.
         libraries = list(run_config.get("libraries", []))
-        python_libraries = {
-            x["pypi"]["package"].split("==")[0].replace("_", "-") for x in libraries if "pypi" in x
-        }
-        for library in ["dagster", "dagster-databricks", "dagster-pyspark"]:
-            if library not in python_libraries:
-                libraries.append(
-                    {"pypi": {"package": "{}=={}".format(library, dagster.__version__)}}
-                )
+        install_default_libraries = run_config.get("install_default_libraries", True)
+        if install_default_libraries:
+            python_libraries = {
+                x["pypi"]["package"].split("==")[0].replace("_", "-")
+                for x in libraries
+                if "pypi" in x
+            }
+            for library in ["dagster", "dagster-databricks", "dagster-pyspark"]:
+                if library not in python_libraries:
+                    libraries.append(
+                        {"pypi": {"package": "{}=={}".format(library, dagster.__version__)}}
```

Comment:
I just realized that, as of dagster 1.0.x, this will not work, as the versions of `dagster-databricks` and `dagster-pyspark` are version 0.16.x, which is not the same as the dagster version.

I can handle this in a follow-up PR, but I just wanted to make sure that this wasn't causing issues for you currently.